### ISSUE TITLE
fallbackLng property in "t" function is ignored

### ISSUE THREAD

**Issue by vgladimir** [i18next/i18next#1111]
Hi,

I have the next setup:
```
i18next.init({
      lng: 'et',
      fallbackLng: 'en',
      defaultNS: 'common',
});
```
Resources:
```
{
    en: {
        hellomm: 'Hello world',
    },
    et: {
        hello: 'Tere maailm',
    },
    ru: {
        hellomm: 'Привет мир',
    }
}
```

Test:
```
i18next.t('hello') -> Tere maailm // Okey
i18next.t('hellomm') -> Hello world // Okey
i18next.t('hellomm', { lng: 'ru' }) -> Привет мир // Okey
i18next.t('hellomm', { fallbackLng: 'ru' }) -> Hello world // Wrong, must be 'Привет мир'
```

The parameter `fallbackLng` is ignored, but by the documentation it must override the default fallback value.

**Comment by jamuhl** [comment]
good catch...passing options.fallbackLng as the second param here: https://github.com/i18next/i18next/blob/master/src/Translator.js#L243 should do the trick.

Will be fixed asap.

**Comment by jamuhl** [comment]
As a temporal workaround `i18next.t('hellomm', { lngs: [i18next.language, 'ru'] })` should have the same effect.

**Comment by jamuhl** [comment]
should be fixed in i18next@11.9.1
